podcast/the-changelog-119.md

```diff
@@ -1,3 +1,6 @@
-* [MEAN.JS](http://meanjs.org/)
-* [roieki (Roie Cohen)](https://github.com/roieki)
-* [amoshaviv (Amos Haviv)](https://github.com/amoshaviv)
\ No newline at end of file
+* [MEAN.JS](http://meanjs.org/) - Open-Source Full-Stack Solution For MEAN Applications!! 
+MEAN.JS is a full-stack JavaScript solution that helps you build fast, robust, and maintainable production web applications using MongoDB, Express, AngularJS, and Node.js. MEAN.JS will help you getting started and avoid useless grunt work and common pitfalls, while keeping your application organized. 
+
+* [roieki (Roie Cohen)](https://github.com/roieki) - GitHub profile of Roie Cohen, co-founder of meanjs.org

```

Comment:
For these we can replace the link text with part of the description, eg:

> * [GitHub profile of Roie Cohen](https://github.com/roieki) - co-founder of meanjs.org

Reply:
done as well! xD